integration_test/crawl_integration_test.go

```diff
@@ -37,7 +37,7 @@ func TestBadVersions(t *testing.T) {
 	}{
 		{name: "single bad version", directory: "../examples/single_bad_version", count: 1, finding: crawl.JarName | crawl.ClassPackageAndName | crawl.ClassFileMd5},
 		{name: "multiple bad versions", directory: "../examples/multiple_bad_versions", count: 13, finding: crawl.JarName | crawl.ClassPackageAndName | crawl.ClassFileMd5},
-		{name: "inside a dist", directory: "../examples/inside_a_dist", count: 2, finding: crawl.JarNameInsideArchive},
+		{name: "inside a dist", directory: "../examples/inside_a_dist", count: 4, finding: crawl.JarNameInsideArchive},
```

Comment:
Should this be catching `6` if we are tripling the number of `$bad_version_name` in the `inside_a_dist` examples dir?

Reply:
As discussed offline, this it right 👌